src/js/api.js

```diff
@@ -288,6 +288,15 @@ export function toPy(obj, { depth = -1 } = {}) {
   return Module.hiwire.pop_value(result);
 }
 
+/**
+ *
```

Comment:
Please add a docstring description, and mention here that the behavior is completely different from the one we had before.

Also it could be good to have an example in the docstring of how to use it, and also mention this is equivalent to `pyodide.runPython('import module; module')` but without adding the module to globals.